### PR TITLE
galaxyside additions for gpu pulsars

### PR DESCRIPTION
**Pull request by cat-bro**
scaffolding for queue vhosts, tpv users and one destination to start with in tpv destinations.yml.j2

### CODE REVIEW

**Comment by cat-bro** [comment]
genome.edu.au needs to be a protected domain for uncontactable users

File: files/galaxy/dynamic_job_rules/production/total_perspective_vortex/destinations.yml.j2 [hunk 1]
```diff
@@ -284,4 +284,25 @@ destinations:
             docker_set_user: '1000'
             docker_memory: '{mem}G'
             docker_sudo: false
+  _pulsar_qld_gpu:
+    inherits: _pulsar_destination
+    max_accepted_cores: 24
+    max_accepted_mem: 209
```
Comment (jlqfab) [inline files/galaxy/dynamic_job_rules/production/total_perspective_vortex/destinations.yml.j2:290]: Where does this number (209) come from?
`free -g` shows that we have a total of 590 G and 584 G available, so 209 would fit 2 of these and still have >100 G remaining.

Comment (jlqfab) [reply files/galaxy/dynamic_job_rules/production/total_perspective_vortex/destinations.yml.j2:290]: additionally, we have 64 cores available.

Comment (cat-bro) [reply files/galaxy/dynamic_job_rules/production/total_perspective_vortex/destinations.yml.j2:290]: 24/209 was mindless pasting ("note to self get actual numbers")

so we need cores: 64 and mem: 582 (can't be bigger than what is set for slurm node)

**Review by jlqfab — APPROVED** [review]
looks good to me

**Pull request merged**